.github/workflows/ci.yml

```
@@ -4,72 +4,81 @@ on:
+
 jobs:
   tests:
+    name: Unit Tests
     runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+      branch: ${{ steps.prereqs.outputs.branch }}
     steps:
+      - name: Prerequisites
+        id: prereqs
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)